prepare

```diff
@@ -24,24 +33,35 @@ echo "Install dependencies"
 set -ex
 
 # TODO: drop once new distros are available
-test "$(lsb_release -cs)" = "jammy" && sudo add-apt-repository -y ppa:v-launchpad-jochen-sprickerhof-de/sbuild
+test $VERSION_CODENAME = "jammy" && sudo apt install -y software-properties-common && sudo add-apt-repository -y ppa:v-launchpad-jochen-sprickerhof-de/sbuild
 sudo apt update
 
+sudo DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends sbuild arch-test mmdebstrap distro-info debian-archive-keyring ccache uidmap curl git-lfs
+
+if [ $DEB_ARCH != $(dpkg --print-architecture) ]; then
+  # use qemu to translate architecture
+  sudo apt install -y --no-install-recommends qemu-user-static binfmt-support
```

Comment:
```suggestion
  sudo apt install -y --no-install-recommends qemu-user-static arch-test
```

Reply:
looks like the jammy version does not even use triggers, so I would propose something like:
```shell
test "$VERSION_CODENAME" = "jammy" && sudo apt install -y --no-install-recommends binfmt-support
sudo apt install -y --no-install-recommends qemu-user-static arch-test
arch-test "$DEB_ARCH" || sudo update-binfmts --enable
```

Reply:
`binfmt-support` is required here as it is a recommended dependency of `qemu-user-static` and I explicitly chose `--no-install-recommends` so it has to be provided manually.

Reply:
It is only needed in jammy that's why I add it later.